src/couchapps/WMStats/_attachments/js/minified/global.min.js

```diff
@@ -66,6 +67,16 @@ WMStats.Globals = function($){
     
     function getReqDetailPrefix () {
         if (_dbVariants[dbname] == "tier1") {
```

Comment:
What is this tier1 dbname? Is it still in use? The same for the analysis one below.